dc/council/code/titles/21/sections/21-309.xml

```diff
@@ -8,17 +8,19 @@
     <para>
       <num>(1)</num>
       <text>An uncertificated security is either:</text>
-      <para>
-        <heading>TRANSFER UNDER THE DISTRICT OF COLUMBIA UNIFORM TRANSFERS TO MINORS ACT</heading>
-        <text>I, __________ [name of transferor or name of representative capacity if a fiduciary] hereby transfer to __________ [name of custodian], as custodian for __________ [name of minor] under the District of Columbia Uniform Transfers to Minors Act, the following: [insert a description of the custodian property sufficient to identify it].</text>
-        <text>Dated: __________</text>
-        <text>________________________</text>
-        <text>[Signature]</text>
-        <text>__________ [name of custodian] acknowledges receipt of the property described above as a custodian for the minor named above under the District of Columbia Uniform Transfers to Minors Act.</text>
-        <text>Dated: __________</text>
-        <text>________________________</text>
-        <text>[Signature of Custodian]</text>
-      </para>
+      <text>
+        <center>
+          <strong>TRANSFER UNDER THE DISTRICT OF COLUMBIA UNIFORM TRANSFERS TO MINORS ACT</strong>
```

Comment:
No change necessary here. Just note how this looks in Division III, pg 291 as a reference point.